README.md

```diff
@@ -164,7 +165,10 @@ async function getApp() {
     });
     const app = fastify({ logger: true });
 
-    app.register(inputValidation.validate());
+    app.register(fastifyUrlData);
+    app.register(inputValidation.validate({
+      skiplist: ['/_metrics']
```

Comment:
If you think that this logic should be applied not just to fastify, I could move the parameter to `init` options instead

Reply:
in general that's a good idea, but as in koa and express you only set the validator on the route i'm not sure that this is a priority

Reply:
yup, that was my line of thinking as well